packages/server/src/tests/epics/choiceCards.ts

```diff
@@ -45,30 +44,29 @@ const buildEpicChoiceCardsTest = (
             paragraphs: paragraphs,
             highlightedText: highlightedText,
             cta: CTAS.control,
-            separateArticleCount: { type: 'above' },
+            separateArticleCount: articlesViewedSettings ? { type: 'above' } : undefined,
```

Comment:
We want to disable the 'above' AC if it's in the copy, so: 
`articlesViewedSettings ? undefined : { type: 'above' }`

Reply:
Thanks @tomrf1!